tests/test_model.py

```diff
@@ -0,0 +1,18 @@
+import os
+import torch
+from tests import _PATH_MODEL
+from src.models.model import DeiTClassifier
+
+
+def test_model():
+    testmodel = DeiTClassifier()
+
+    model_path = os.path.join(_PATH_MODEL, "saved_models", "model.pt")
```

Comment:
We do not store model weights on github. Here we can initialize the model randomly - we are only looking at the shape not the actual output.